hpedockerplugin/volume_manager.py

```diff
@@ -119,9 +119,10 @@ def _initialize_configuration(self):
                         "Failed to initialize driver - cpg_map not defined for"
                         "replication device")
 
-                self.tgt_bkend_config.hpe3par_snapcpg = \
-                    self._extract_remote_cpgs(
-                        self.tgt_bkend_config.snap_cpg_map)
+                if self.tgt_bkend_config.snap_cpg_map:
```

Comment:
@bhagyashree-sarawate As per https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/active-passive-based-replication.md both _cpg_map_ and _snap_cpg_map_ in replication_device section are mandatory. 

But as per code, when user doesn't give _snap_cpg_map_, cpg_map is used as snap_cpg_map.

Please fix the documentation for this.

Reply:
done